MAUI/ImageEditor/Crop.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title: Crop in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about Crop support in Syncfusion .NET MAUI ImageEditor (SfImageEditor) control and more.
+platform: MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Crop in .NET MAUI ImageEditor (SfImageEditor)
+
+You can use the cropping tool in the Image Editor to select and crop a specific portion of an image.
```

Comment:
Using the Image Editor's cropping tool, you can select and crop a particular section of an image.